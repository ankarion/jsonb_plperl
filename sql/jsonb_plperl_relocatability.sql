CREATE EXTENSION jsonb_plperl CASCADE;
CREATE SCHEMA test;
alter extension jsonb_plperl set schema test;
create function test.test(val jsonb) returns jsonb
language plperl
transform for type jsonb
as $$
return val
$$;

select test.test('1'::jsonb);

drop extension plperl cascade;
drop schema test cascade;
