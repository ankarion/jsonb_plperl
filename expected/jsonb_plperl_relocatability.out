CREATE EXTENSION jsonb_plperl CASCADE;
NOTICE:  installing required extension "plperl"
CREATE SCHEMA test;
alter extension jsonb_plperl set schema test;
create function test.test(val jsonb) returns jsonb
language plperl
transform for type jsonb
as $$
return val
$$;
select test.test('1'::jsonb);
 test  
-------
 "val"
(1 row)

drop extension plperl cascade;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to extension jsonb_plperl
drop cascades to function test.test(jsonb)
drop schema test cascade;
